template_avancado/category.html

```diff
@@ -181,18 +181,28 @@
 		{% endif %}
 			<div class="row">
 				<div class="col-xs-12">
-					{# Titulo da Página #}
+					{# Titulo da P�gina #}
 					{% if term is not empty %}
 						<h1 class="category-header">{{ "Resultado da busca por"|trans({'en':'Results with'}) }} {{ term }}</h1>
 					{% else %}
 						<h1 class="category-header">{{ category.name }}</h1>
+						
+						{# Imagem da categoria #}
+    				    {% if category.image is not empty %}
+        					<div class="category-img">
+        						{{ category_img(category, 'full', true) }}
+        					</div>
+        				{% endif %}
+					    
+					    {# Descri��o da categoria #}
+					    {% if category.excerpt is not empty %}
+    						<div style="margin: 10px 0px 0px;">{{ category.excerpt }}</div>
```

Comment:
Evita usar css inline! Sempre que puder cria uma classe. Se for coisas genérica como margin-top, cria uma para isso. tipo .margin-top-10